dev/io.openliberty.microprofile.openapi.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -23,4 +23,16 @@ docPath=MicroProfile OpenAPI document endpoint path
 docPath.desc=Specifies the URL path for obtaining OpenAPI documents. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.).
 
 uiPath=MicroProfile OpenAPI UI endpoint path
-uiPath.desc=Specifies the URL path for accessing the MicroProfile OpenAPI UI. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.) If no value is specified, the path is set to $docPath/ui.
\ No newline at end of file
+uiPath.desc=Specifies the URL path for accessing the MicroProfile OpenAPI UI. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.) If no value is specified, the path is set to $docPath/ui.
+
+includeApp=MicroProfile OpenAPI included application.
+includeApp=An application, one per element, that are to be included in the OpenAPI document. The application name is as defined in the server.xml, with a default name of the archive filename excluding any extension.  The "all" special value includes all available applications. 
+
+excludeApp=MicroProfile OpenAPI excluded modules.
+excludeApp=An application, one per element, that are to be included in the OpenAPI document. The application name is as defined in the server.xml, with a default name of the archive filename excluding any extension.  The "all" special value excludes all applications.
+
+includeModule=MicroProfile OpenAPI included modules.
+includeModule=A module, one per element, that are to be included in the OpenAPI document. Module names are in the format {ApplicationName}/{ModuleName}. ModuleName is the Jakarta EE module name and is logged under the message SRVE0169I. 
+
+excludeModule=MicroProfile OpenAPI excluded modules.
+excludeModule=A  module, one per element, that are to be included in the OpenAPI document. Module names are in the format {ApplicationName}/{ModuleName}. ModuleName is the Jakarta EE module name and is logged under the message SRVE0169I.
```

Comment:
```suggestion
includeApplication=MicroProfile OpenAPI included applications
includeApplication.desc=Application names, one per element, that are to be included in the OpenAPI document. The name of an application may be defined in the server.xml, otherwise it defaults to the application archive filename excluding any extension.  The "all" special value includes all available applications. 

excludeApplication=MicroProfile OpenAPI excluded applications
excludeApplication.desc=Application names, one per element, that are to be excluded from the OpenAPI document.

includeModule=MicroProfile OpenAPI included modules
includeModule.desc=Module names, one per element, that are to be included in the OpenAPI document. Module names must be provided in the format {ApplicationName}/{ModuleName}. The name of a module may be defined in its deployment descriptor, otherwise it defaults to the module archive filename excluding any extension. It is also logged in the SRVE0169I message.

excludeModule=MicroProfile OpenAPI excluded modules
excludeModule.desc=Module names, one per element, that are to be excluded from the OpenAPI document.
```

The keys in this file must match the metatype.xml.

I've reworded the messages a bit and fixed some cases where it said included vs. excluded, though I'm sure ID will have some more suggestions.

I'm not sure whether we should add to this:
- default values
- the versions of mpOpenAPI that this config is valid for